README.md

```diff
@@ -3,11 +3,11 @@ This repository contains ATF scripts and data to run it.
 
 ## How To:
 
-* [Setup SDL](https://github.com/smartdevicelink/sdl_atf/blob/develop/README.md). 
+* [Setup SDL](https://github.com/smartdevicelink/sdl_core/blob/master/README.md). 
```

Comment:
@LuxoftAKutsan, I think we should use refs to the whole repo. In this case we don't have to operate with branches